cursorless-talon/src/cheatsheet/sections/scopes.py

```diff
@@ -10,6 +10,7 @@ def get_scopes():
             {
                 "argumentOrParameter": "Argument",
                 "boundedNonWhitespaceSequence": "Non whitespace sequence stopped by surrounding pair delimeters",
+                "boundedParagraph": "Paragraph stopped by surrounding pair delimeters",
```

Comment:
I know it's being used by `boundedNonWhitespaceSequence` already, but "stopped by" is kind of awkward to my ears. On the other hand it's tricky because of the brevity needed for the cheat sheet. Maybe we can brainstorm a bit in the meetup.

Reply:
Also not a fan of that phrasing. If we find something better we will update it in both places of course.